pyanaconda/ui/gui/spokes/lib/resize.py

```diff
@@ -368,21 +368,20 @@ def _setup_slider(self, min_size, max_size, default_size):
 
     def _update_action_buttons(self, row):
         obj = PartStoreRow(*row)
-        device_name = obj.name
-        device_data = DeviceData.from_structure(
-            self._device_tree.GetDeviceData(device_name)
-        )
 
-        # Disks themselves may be editable in certain ways, but they are never
-        # shrinkable.
         self._preserve_button.set_sensitive(obj.editable)
-        self._shrink_button.set_sensitive(obj.editable and not device_data.is_disk)
+        self._shrink_button.set_sensitive(obj.editable)
```

Comment:
Why we don't have to disable shrink button for disk anymore?

Reply:
It happens a few lines below. See: `self._shrink_button.set_sensitive(is_resizable)`